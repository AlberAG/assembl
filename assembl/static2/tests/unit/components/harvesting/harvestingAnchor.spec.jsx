import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import HarvestingAnchor from '../../../../js/app/components/harvesting/harvestingAnchor';

describe('harvestingAnchor component', () => {
  it('should match harvestingAnchor snapshot', () => {
    const props = {
      harvestingMenuPosition: 100
    };
    const shallowRenderer = new ShallowRenderer();
    shallowRenderer.render(<HarvestingAnchor {...props} />);
    const result = shallowRenderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});