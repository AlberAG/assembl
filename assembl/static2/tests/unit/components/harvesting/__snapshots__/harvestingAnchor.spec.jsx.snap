// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`harvestingAnchor component should match harvestingAnchor snapshot 1`] = `
<div
  className="harvesting-anchor"
  style={
    Object {
      "marginTop": "undefinedpx",
    }
  }
>
  <div
    className="left-side-harvesting-button"
  >
    <div
      className="left-side-harvesting-button__label"
    >
      <Translate
        value="harvesting.harvestSelection"
      />
    </div>
    <div
      className="left-side-harvesting-button__button"
      onClick={undefined}
      onMouseDown={undefined}
    >
      <div
        className="left-side-harvesting-button__button__inside"
      >
        <span
          className="confirm-harvest-button assembl-icon-catch"
        />
      </div>
    </div>
  </div>
</div>
`;
